pkg/daemon/update_test.go

```diff
@@ -136,10 +136,11 @@ func TestReconcilable(t *testing.T) {
 	checkReconcilableResults(t, "Filesystem", isReconcilable)
 
 	// Verify Raid changes react as expected
+	var stripe = "stripe"
 	oldIgnCfg.Storage.Raid = []ign3types.Raid{
 		{
 			Name:    "data",
-			Level:   "stripe",
+			Level:   &stripe,
```

Comment:
looks fine but just interested why the change here?

Reply:
The ignition spec changed -- not my choice. :smile: 

 Sometimes ignition changes what is/isn't a pointer between versions. They try not to, but it happens. The ignition parsing (up) and the ignition converter (down) usually handle this for us (although sometimes it still doesn't line up exactly right, e.g.:  https://github.com/coreos/ign-converter/pull/47)